client/webserver/locales/parse.go

```diff
@@ -2,7 +2,7 @@ package locales
 
 import "regexp"
 
-var keyRegExp = regexp.MustCompile(`\[\[\[([a-zA-Z0-9 _.-:]+)\]\]\]`)
+var keyRegExp = regexp.MustCompile(`\[\[\[([a-zA-Z0-9 _.\-:]+)\]\]\]`)
```

Comment:
Good catch.  With the `:` at the end it (added last week), it must have started interpreting the `-` to mean a character range.

https://regex101.com/r/aDB2hS/1

![image](https://user-images.githubusercontent.com/9373513/197884840-f242e531-0a22-4f12-bb0b-d797339beaa8.png)

![image](https://user-images.githubusercontent.com/9373513/197885006-ccf8f768-c215-49df-beda-124aea42f300.png)

Reply:
> https://regex101.com/r/aDB2hS/1

phew, didn't notice the new interpretation. Good catch.

Reply:
I don't think all those rules can (or should) be remembered : ) that's what tests are for

Reply:
Indeed.  Thanks for the harder pattern.  Now we do test all the rules.

Reply:
However, we don't have a negative test (not matching when something not in the intended set is included), which would have caught that this was being treated as a range.

Reply:
Great, thanks for adding a negative test that does include the triple brackets.  The error prone bits are clearly what's inside the brackets.